installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -0,0 +1,135 @@
+include::modules/installation-gcp-permissions.adoc[leveloffset=+3]
+include::modules/installation-gcp-regions.adoc[leveloffset=+2]
+include::modules/installation-gcp-install-cli.adoc[leveloffset=+2]
+
+include::modules/installation-gcp-user-infra-config-host-project-vpc.adoc[leveloffset=+1]
+include::modules/installation-gcp-dns.adoc[leveloffset=+2]
+include::modules/installation-creating-gcp-vpc.adoc[leveloffset=+2]
+include::modules/installation-deployment-manager-vpc.adoc[leveloffset=+3]
+
+include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
+
+include::modules/installation-initializing-manual.adoc[leveloffset=+2]
+
+include::modules/installation-gcp-user-infra-shared-vpc-config-yaml.adoc[leveloffset=+2]
+
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
+//include::modules/installation-three-node-cluster.adoc[leveloffset=+2]
+
+include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+2]
+.Additional resources
+
+[id="installation-gcp-user-infra-exporting-common-variables-vpc"]
+== Exporting common variables
+
+include::modules/installation-extracting-infraid.adoc[leveloffset=+2]
+include::modules/installation-user-infra-exporting-common-variables.adoc[leveloffset=+2]
+
+include::modules/installation-creating-gcp-lb.adoc[leveloffset=+1]
+include::modules/installation-deployment-manager-ext-lb.adoc[leveloffset=+2]
+include::modules/installation-deployment-manager-int-lb.adoc[leveloffset=+2]
+
+include::modules/installation-creating-gcp-private-dns.adoc[leveloffset=+1]
+include::modules/installation-deployment-manager-private-dns.adoc[leveloffset=+2]
+
+include::modules/installation-creating-gcp-firewall-rules-vpc.adoc[leveloffset=+1]
+include::modules/installation-deployment-manager-firewall-rules.adoc[leveloffset=+2]
+
+include::modules/installation-creating-gcp-iam-shared-vpc.adoc[leveloffset=+1]
+include::modules/installation-deployment-manager-iam-shared-vpc.adoc[leveloffset=+2]
+
+include::modules/installation-gcp-user-infra-rhcos.adoc[leveloffset=+1]
+
+include::modules/installation-creating-gcp-bootstrap.adoc[leveloffset=+1]
+include::modules/installation-deployment-manager-bootstrap.adoc[leveloffset=+2]
+
+include::modules/installation-creating-gcp-control-plane.adoc[leveloffset=+1]
+include::modules/installation-deployment-manager-control-plane.adoc[leveloffset=+2]
+
+include::modules/installation-gcp-user-infra-wait-for-bootstrap.adoc[leveloffset=+1]
+
+include::modules/installation-creating-gcp-worker.adoc[leveloffset=+1]
+include::modules/installation-deployment-manager-worker.adoc[leveloffset=+2]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/installation-approve-csrs.adoc[leveloffset=+1]
+
+include::modules/installation-gcp-user-infra-adding-ingress.adoc[leveloffset=+1]
+
+[id="installation-gcp-user-infra-vpc-adding-firewall-rules"]
```

Comment:
Per upstream doc, there are 2 approaches to add firewall rules: 1 is Adding firewall rules based on cluster events and another is Adding a cluster-wide firewall rules. So I suggest to change the section name as below:
Adding the ingress firewall rules
    Adding firewall rules based on cluster events
    Adding cluster-wide firewall rules (alternative)

Reply:
I checked with the author of the dev docs, and dev's preference is to use the event-based rules to better mimic the IPI functionality.  I structured the docs to mimic that.

Reply:
I see. What I mean was that we need to instruct that Adding cluster-wide firewall rules is an alternative of event-based rules, if there is any issue on event, user can add cluster-wide firewall rules instead.